src/components/pages/Photos/elements/cameraNavbar/CameraNavbar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import RoverCameraButton from './RoverCameraButton';
+import './CameraNavbar.css';
+
+export default function Navbar() {
+  return (
+    <ul className="camera-navbar">
```

Comment:
I think that now the child elements are `<button>` rather than `<li>` elements we can change this to a normal `<div>` tag.